src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTokenTextCheck.java

```diff
@@ -71,6 +77,15 @@
  *   &lt;property name=&quot;ignoreCase&quot; value=&quot;true&quot;/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public void myTest() {
+ *     int test1 = 0; // ok
```

Comment:
there is `NUM_LONG` in token list, but there is nothing in example, please add

Reply:
ok, I am done.

Reply:
and update the first post~